test_regress/t/t_trace_two_dumpfst_cc.out

```diff
@@ -83,5 +84,5 @@ b00000000000000000000000000001010 "
 0!
 #110
 1!
-b00000000000000000000000000001011 "
 b00000000000000000000000000001100 #
+b00000000000000000000000000001011 "
```

Comment:
vcddiff seems to have a bug as it will fail without swapping these last 2 lines, which should make zero differnce. It claims it cannot find the `"` signalin the reference file if it appears in the original order.